File: DictionaryProject/Commands/HighlightAttribute.vb
<AttributeUsage(AttributeTargets.Class, AllowMultiple:=True)> _
Public Class HighlightAttribute
    Inherits Attribute

    Private _Level As Integer = 1
    Public Property Level() As Integer
        Get
            Return _Level
        End Get
        Set(ByVal value As Integer)
            _Level = value
        End Set
    End Property

    Private _RegEx As String
    Public Property RegEx() As String
        Get
            Return _RegEx
        End Get
        Set(ByVal value As String)
            _RegEx = value
        End Set
    End Property

    Private _Color As String
    Public Property Color() As String
        Get
            Return _Color
        End Get
        Set(ByVal value As String)
            _Color = value
        End Set
    End Property

    Private _NextLevel As Integer = -1
    Public Property NextLevel() As Integer
        Get
            Return _NextLevel
        End Get
        Set(ByVal value As Integer)
            _NextLevel = value
        End Set
    End Property
End Class

File: monowordbuilder/wordbuilderbase/Commands/HighlightAttribute.cs
using System;

namespace Whee.WordBuilder.Project.Commands
{
	[AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
	public class HighlightAttribute : Attribute
	{
	
		private int _Level = 1;
		public int Level {
			get { return _Level; }
			set { _Level = value; }
		}
	
		private string _RegEx;
		public string RegEx {
			get { return _RegEx; }
			set { _RegEx = value; }
		}
	
		private string _Color;
		public string Color {
			get { return _Color; }
			set { _Color = value; }
		}
	
		private int _NextLevel = -1;
		public int NextLevel {
			get { return _NextLevel; }
			set { _NextLevel = value; }
		}
	}
}